simpleenergy/bughouse-ranking
# Piper/track black and white overall rankings

### What is the problem / feature ?

- It'd be cool to know if your rating differed much depending on whether you played as black or as white.

### How did it get fixed / implemented ?

- Add independent tracking of player ratings as black and as white.
- @nathanlubchenco what do you think about how this was implemented?  Is this approach sound?

### How can someone test / see it ?

- run `./manage.py recompute_ratings` to re-run all ratings engines across all games.
- The `/player/:id/ratings/` endpoints should return all ratings so querying those will give you ratings with the new `overall:black` and `overall:white` ratings.

*Here is a cute animal picture for your troubles...*

![l-unlikely-friends](https://cloud.githubusercontent.com/assets/824194/6169137/b78cb594-b288-11e4-9a0e-bf7c9efe92ac.jpeg)
